inference-engine/src/transformations/src/ngraph_ops/framework_node.cpp

```diff
@@ -25,31 +25,73 @@ shared_ptr<Node> op::FrameworkNode::clone_with_new_inputs(const OutputVector& ne
     return node;
 }
 
+void op::FrameworkNode::cache_output_descriptor() {
+    for (uint64_t i = 0; i < get_output_size(); ++i) {
```

Comment:
if I remember right `get_output_size()` returns `size_t`.
```suggestion
    for (size_t i = 0; i < get_output_size(); ++i) {
```

Reply:
no problem :)